lib/src/androidTest/kotlin/at/bitfire/ical4android/ICalPreprocessorTest.kt

```diff
@@ -16,11 +18,17 @@ import org.junit.Assert.assertEquals
 import org.junit.Test
 import java.io.InputStreamReader
 import java.io.StringReader
+import org.junit.Assume.assumeTrue
 
 class ICalPreprocessorTest {
 
     @Test
     fun testPreprocessStream_appliesStreamProcessors() {
+        assumeTrue(
+            "Can only run on API Level 24 or newer because mockkObject doesn't support Android < P (SDK 28)",
+            Build.VERSION.SDK_INT >= 28
+        )
+
```

Comment:
As far as I remember, there's an extra annotation for that: [`@SdkSuppress` should do it](https://developer.android.com/training/testing/instrumented-tests/androidx-test-libraries/runner) :)

Reply:
I've now switched to `SdkSuppress`, but left a comment just that we know why in the future.